sa/saro.go

```diff
@@ -1353,3 +1353,21 @@ func (ssa *SQLStorageAuthorityRO) GetMaxExpiration(ctx context.Context, req *emp
 func (ssa *SQLStorageAuthority) GetMaxExpiration(ctx context.Context, req *emptypb.Empty) (*timestamppb.Timestamp, error) {
 	return ssa.SQLStorageAuthorityRO.GetMaxExpiration(ctx, req)
 }
+
+// Health implements the grpc.checker interface.
+func (ssa *SQLStorageAuthority) Health(ctx context.Context) error {
```

Comment:
A scenario would be that an SA calling one of the read-only methods, if that SA had a config for a mariadb user that was missing the `SELECT` grant on the mariadb server, then the healthcheck would fail for that SARO embedded in that SA.

Reply:
Not sure how you feel about this, but I've updated this to call `.Health()` on the embedded SQLStorageAuthorityRO.